waiter/test/waiter/service_description_test.clj

```diff
@@ -1651,7 +1651,11 @@
     (is (nil? (kv/fetch cache-kv-store (service-id->key service-id-3))))
     (is (= (service-id->service-description service-id-4) (kv/fetch cache-kv-store (service-id->key service-id-4))))
 
-    (refresh-service-descriptions cache-kv-store #{service-id-1 service-id-2 service-id-3 service-id-4})
+    (let [service-ids #{service-id-1 service-id-2 service-id-3 service-id-4}]
```

Comment:
I _really_ dislike threading into macros like `(is)`. I think this is much easier to read:
```
    (let [service-ids #{service-id-1 service-id-2 service-id-3 service-id-4}
          service-ids-in (conj service-ids "service-id-unknown1" "service-id-unknown2")
          service-ids-out (refresh-service-descriptions cache-kv-store service-ids-in)]
      (is (= service-ids service-ids-out)))
```

Reply:
Done.